libsolidity/codegen/ir/IRGenerator.cpp

```diff
@@ -73,10 +76,21 @@ pair<string, string> IRGenerator::run(ContractDefinition const& _contract)
 	return {warning + ir, warning + asmStack.print()};
 }
 
-string IRGenerator::generate(ContractDefinition const& _contract)
+string IRGenerator::generate(
+	ContractDefinition const& _contract,
+	map<ContractDefinition const*, string const&> const& _otherYulSources
```

Comment:
Is a map to a const& a valid type?

Reply:
Hm, it is tricky, yes. I can remove reference and be sure about the type here